### DISCUSSION TITLE
Minor bug: Theme temporarily resets when opening some settings tab

### DISCUSSION THREAD

**Discussion by rafern** [community/community#22219] [Accessibility]
<p>When selecting one of the tabs in the Billing &amp; plans settings, the theme resets to light mode. This isn’t persistent and the theme works correctly after reloading the browser tab.<br>
Screen capture:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/a/2a90d97c74fd165ca90cae67b046ee92c0409574.gif" data-download-href="/uploads/short-url/64yokLHGfENgP6wYVdUYZPMezac.gif?dl=1" title="Peek 2020-12-08 18-27" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15365765/181069602-7b2dc60d-c779-4feb-be27-166eb05c8c4a.gif" alt="Peek 2020-12-08 18-27" data-base62-sha1="64yokLHGfENgP6wYVdUYZPMezac" width="690" height="177" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/a/2a90d97c74fd165ca90cae67b046ee92c0409574_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Peek 2020-12-08 18-27</span><span class="informations">1017×262 110 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Edit: someone reported the same bug 2 minutes later lol: <a href="https://github.community/t/bug-report-of-dark-mode-beta/147862" class="inline-onebox">Bug report of Dark Mode Beta</a></p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for reporting this <a class="mention" href="/rafern">@rafern</a>! This one somehow slipped through the cracks but I’ve added it to our backlog to get fixed. Appreciate you taking the time to share the feedback here and let us know if you come across any other bugs. Otherwise, hope you’re enjoying dark mode!</p>

**Comment by cehfisher** [comment]
Hi @rafern - after reviewing your feedback against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
